### ISSUE TITLE
Comparing Different models

### ISSUE THREAD

**Issue by mohammadshahabuddin** [koutilya-pnvr/SharinGAN#5]
Hi,
Could you please tell me how you compared different models? Did you use the same learning rate, number of epochs, Number of decay epochs, image size, optimizer among all models? Also, did you collect test results using the final saved generator or did you use the best results testing all saved generators at different epochs?

**Comment by koutilya-pnvr** [comment]
Hi @mohammadshahabuddin , thanks for your interest in our work. We trained the network for a specific number of iterations (150000 for the monocular depth estimation) and picked the model that yielded the best validation performance on the validation split and used that model to report the test performance. We believe that's how a machine learning model should be evaluated. 

The comparison with other works was done on similar hyperparameters and their reported best test accuracy in their papers.

Let us know in case you have any more questions.
